src/geo/map-view.js

```diff
@@ -121,6 +121,13 @@ var MapView = View.extend({
     return this;
   },
 
+  _linkMapHelpers: function () {
+    this.map.set({
```

Comment:
Why set these as attributes? Here's an idea:

```
this.map.setPixelToLatLngConverter(this.containerPointToLatLng.bind(this));
this.map.setLatLngToPixelConverter(this.latLngToContainerPoint.bind(this));
```

```
// in map.js
setPixelToLatLngConverter: function (pixelToLatLngConverter) {
  this._pixelToLatLngConverter = pixelToLatLngConverter;
},

setLatLngToPixelConverter: function (latLngToPixelConverter) {
  this._latLngToPixelConverter = latLngToPixelConverter;
},

pixelToLatLng: function (point) {
  return this._pixelToLatLngConverter && this._pixelToLatLngConverter(point);
},

latLngToPixel: function (latLng) {
  return this._latLngToPixelConverter && this._latLngToPixelConverter(point);
}

```

This way, those methods will be exposed directly. They could raise an exception perhaps if "converters" haven't been set... Just an idea...